src/locales/en/pages.yaml

```diff
@@ -732,60 +732,50 @@ addendum-50:
-      When we near the summer holidays there is often a larger project. This year we went to Paris and Tours (France). Next year we will have the 11th lustrum project which promises to be an amazing experience.
-      All these different activities could not take place without the hard work of multiple committees. Being part of a committee is a fun way to get to know people and work on personal development which is strongly encouraged within the association. This way you get to know a lot of other members quickly.
-
-    ## What are we looking for?
-      We are looking for people who enjoy making music in small or larger groups. This can be done by singing in the choir, participating in chamber music events or by playing in the orchestra.
-      In the choir everybody is allowed to participate. No audition is required, however having fun is a must!
-
-      For the orchestra there are auditions. At the moment we are looking for the following instruments:
-      - Violin
-      - Viola
-      - Cello
-      - Double basses
-      - Horn
-      - Flute (1 permanent spot and one temporary spot until February)
-      - Oboe
-      - Bassoon
-      - Trumpet (1 temporary spot until February)
-
-    ## What’s on the program for this year?
-      In the first large project of 2022-2023, the orchestra will play Mahler's 5th symphony and the choir will sing the Requiem by Fauré. Moreover, both the choir and orchestra will be the backing of the boyband FOURCE during a Christmas concert. A smaller side project for the first semester is of an educational nature: a chamber orchestra will give two shows of Peter and the Wolf by Prokofiev together with a storyteller.
-      Since it will be our 11th lustrum coming year, we will work on a grand lustrum project in the second semester. It will consist of a series of 5 concerts that will sparkle the audience’s imagination by combining nature, technical innovations, choir, orchestra and chamber music.
+    Besides the regular rehearsals we have a rehearsal weekend twice a year. During these weekends we work hard on the current projects as well as party and have fun.  
 
-    ## Is there a place to do some private practicing?
-      As a member of Krashna Musika we will automatically provide you with an X membership (12 months). This means you will be able to use the facilities of the sports and culture centre. Here one can also find rehearsal spots for if you do not wish to disturb your roommates too much.
+    Krashna is not only a music association, but switches into a sailing association once every two years. Other sport activities are practiced by our members such as soccer and beach volleyball. For instance, Krashna participates in the annual inter-orchestral soccer day. 
 
+    Every two years we go on tour. Next summer in 2024 we will be going to Berlin and Krakow. This is something you don't want to miss!
+    ## Who are we looking for?
```

Comment:
Why not list the instrument spots we have? Just because we don't know yet? If that's the reason, let's merge this and try to update when we know.